lib/mix/lib/mix/tasks/test.coverage.ex

```diff
@@ -228,6 +228,14 @@ defmodule Mix.Tasks.Test.Coverage do
     {module_results, totals} = gather_coverage(results, keep)
     module_results = Enum.sort(module_results, :desc)
     print_summary(module_results, totals, summary_opts)
+
+    case totals < threshold(summary_opts) do
+      true ->
+        exit({:shutdown, 1})
+
+      false ->
+        :ok
+    end
```

Comment:
We don't want to fail immediately because if you are in an umbrella, you want to show the coverage of all umbrella apps and then exit. So instead please do this:

```suggestion
    if totals < threshold(summary_opts) do
      System.at_exit(fn _ -> exit({:shutdown, 1}) end)
    end

    :ok
```